json-schemas/v1/activity/index.json

```diff
@@ -32,9 +62,6 @@
 		"endTime": {
 			"type": "string"
 		},
-		"parentId": {
-			"type": "string"
-		},
 		"waitReason": {

```

Comment:
Did we move away from creating an enum for waitReasons? I know it was mentioned in one of the chats.

Reply:
Got dragged in to looking at payments service stuff. will add that now.

Reply:
Okay great thanks.